packages/pie-toolbox/src/code/charting/chart.jsx

```diff
@@ -70,6 +70,8 @@ export class Chart extends React.Component {
     changeInteractiveEnabled: PropTypes.bool,
     changeEditableEnabled: PropTypes.bool,
     language: PropTypes.string,
+    mathMlOptions: PropTypes.object,
+    charactersLimit: PropTypes.number,
```

Comment:
I suggest to rename this prop to `labelsCharactersLimit` so that the prop to be more suggestive of what is being used for. In the future we may want to add limitations for other inputs as well

Reply:
Done